js/script.js

```diff
@@ -105,4 +105,114 @@ const nav = document.querySelector(".nav"),
                 hiddenTimelineItems.classList.add('hidden');
                 readMoreButton.textContent = 'Read More';
             }
-            });
\ No newline at end of file
+            });
+
+            /* ================================== chatbot ================================== */
+
+            const chatBody = document.querySelector(".chat-body");
+            const messageInput = document.querySelector(".message-input");
+            const sendMessageButton = document.querySelector("#send-message");
+
+            // API setup
+            const API_KEY = "AIzaSyC3MJvbzNyNoJirWTcUAzN3zrhzugmvkTc";
+            const API_URL = `https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash:generateContent?key=${API_KEY}`;
```

Comment:
Try also to add the Dialog flow So the chatbot can be able to answer questions when it it comes to your personal portfolio